vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -324,7 +355,14 @@ def forward(
                 # triton attention
                 # When block_tables are not filled, it means q and k are the
                 # prompt, and they have the same length.
+                att_masks = None
```

Comment:
nit: Let's use `attn_masks` for consistency?
```suggestion
                attn_masks = None
```

Reply:
Thanks, fixed.